transaction/core/src/tx.rs

```diff
@@ -225,11 +226,13 @@ impl TxPrefix {
 /// An "input" to a transaction.
 #[derive(Clone, Deserialize, Eq, PartialEq, Serialize, Message, Digestible)]
 pub struct TxIn {
-    /// A "ring" of outpuuts containing the single output that is being spent.
+    /// A "ring" of outputs containing the single output that is being spent.
+    /// It would be nice to use [TxOut; RING_SIZE] here, but Prost only works with Vec.
```

Comment:
I was really hoping to use arrays here so that the type system would enforce things like "ring and proofs must have the same length". Bummer.